sga/templatetags/math.py

```diff
@@ -1,14 +1,21 @@
+"""
+Custom math templatetags
+"""
+
 from django import template
 
 register = template.Library()
 
+
 @register.filter
 def subtract(value, arg):
-    """ Subtracts the arg from the value """
+    """
+    Subtracts the arg from the value
+    """
     try:
         return int(value) - int(arg)
     except (ValueError, TypeError):
         try:
             return value - arg
-        except Exception:
-            return ""
\ No newline at end of file
+        except Exception:  # pylint: disable=broad-except
+            return ""
```

Comment:
yes, but the `+` has a meaning even if you "add" lists or strings. For the `-` you cannot do the same, so it does not make much sense the part after the first `except`.